src/bundle/DependencyInjection/Compiler/ViewBuilderRegistryPass.php

```diff
@@ -28,19 +30,21 @@ class ViewBuilderRegistryPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        if (
-            !$container->hasDefinition(self::VIEW_BUILDER_REGISTRY)
-            || !$container->hasDefinition(self::VIEW_BUILDER_CONTENT_TRANSLATE)
-        ) {
+        if (!$container->hasDefinition(self::VIEW_BUILDER_REGISTRY)) {
             return;
         }
 
-        $registry = $container->findDefinition(self::VIEW_BUILDER_REGISTRY);
-
-        $viewBuilders = [
-            $container->getDefinition(self::VIEW_BUILDER_CONTENT_TRANSLATE),
-        ];
+        $viewBuilders = [];
+        if ($container->hasDefinition(self::VIEW_BUILDER_CONTENT_TRANSLATE)) {
+            $viewBuilders[] = $container->getDefinition(self::VIEW_BUILDER_CONTENT_TRANSLATE);
+        }
+        if ($container->hasDefinition(self::VIEW_BUILDER_RELATION)) {
+            $viewBuilders[] = $container->getDefinition(self::VIEW_BUILDER_RELATION);
+        }
 
-        $registry->addMethodCall('addToRegistry', [$viewBuilders]);
+        if ($viewBuilders) {
+            $registry = $container->findDefinition(self::VIEW_BUILDER_REGISTRY);
+            $registry->addMethodCall('addToRegistry', [$viewBuilders]);
```

Comment:
This is out of PR scope: Does type of `$viewBuilders` should be `\Symfony\Component\DependencyInjection\Reference[]` instead of `\Symfony\Component\DependencyInjection\Definition[]`.